source/server/http/admin.h

```diff
@@ -135,7 +135,7 @@ class AdminImpl : public Admin,
                       Buffer::Instance& response);
   static std::string statsAsJson(const std::map<std::string, uint64_t>& all_stats,
                                  const std::vector<Stats::ParentHistogramSharedPtr>& all_histograms,
-                                 bool pretty_print = false);
+                                 bool used_only, bool pretty_print = false);
```

Comment:
nit: this isn't really a comment on your PR since you're just being consistent with the preexisting code, but is it common to add `const` to the function definition alone at the cost of making the declaration and definition slightly inconsistent?  Furthermore, is it typical to make passed-by-value primitives `const` in the first place?  IMO, functions are really contracts with the caller, and codifying `const` primitives into that seems a little restrictive.